example/server.js

```diff
@@ -2,22 +2,44 @@
 
 var Hapi = require('hapi');
 var Hoek = require('hoek');
-var tags = require('../lib/index.js');
+var fs = require('fs');
 
 var server = new Hapi.Server();
+var tagsData = JSON.parse(fs.readFileSync('./example/tags.json').toString());
+var tags = require('../lib/index.js');
+var tagsPool = require('./pg.js');
+
+function init (port, pgConfig, callback) {
+  server.connection({ port: port });
+
+  server.register([{
+    register: tags,
+    options: { tags: tagsData, tagsPool: tagsPool(pgConfig) }
```

Comment:
For now it's working fine because we just have one plugin but we might need to create a variable which contain the pool that you can reuse on other plugin later on. I don't think we want to create a pool for each plugin.